### PR TITLE
Fixed spelling

### PR DESCRIPTION
**Pull request by Mailaender**
> Like most names, it should be written in lower case with an initial capital, that is, "Lua". Please do not write it as "LUA", which is both ugly and confusing, because then it becomes an acronym with different meanings for different people. So, please, write "Lua" right!

### CODE REVIEW

File: docs/about-mono/languages/index.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Object Pascal
 
 Their [Delphi Prism](http://www.codegear.com/products/delphi/prism) compiler support Mono out of the box.
 
-LUA
+Lua
 ---
 
 [Lua2Il](http://www.lua.inf.puc-rio.br/luanet/lua2il/) is a compiler that will allow you to run your existing Lua code or reuse the existent expertise you have on Lua in your application and run it with the Mono JIT compiler.
```
Comment (akoeplinger) [inline docs/about-mono/languages/index.md]: The link doesn't work anymore, do you know if Lua2IL still exists? If not, maybe it's better to remove it here completely.

Comment (Mailaender) [inline docs/about-mono/languages/index.md]: Fixed the link. Never worked with it though.

Comment (Mailaender) [inline docs/about-mono/languages/index.md]: Pinging @mascarenhas

**Comment by txdv** [comment]
LUA is such a nice language

**Comment by Mailaender** [comment]
> ~~LUA~~ is such a nice language
> Lua is such a nice language

SCNR

**Comment by akoeplinger** [comment]
I'm going to merge this, if we're hearing back from @mascarenhas that the project is dead we can always remove it then :smile:

**Comment by txdv** [comment]
The link is kinda dead, just remove it.

**Comment by mascarenhas** [comment]
Lua2IL is a dead project, but there are several active Lua bindings and implementations by @NLua. You might want to link to them. :-)

**Comment by akoeplinger** [comment]
@mascarenhas Thanks for the update, added it in f7f2c8c2ea7b657851bffdd3bffe5d43c5dbce64.

**Pull request merged**